cmd/oci-csi-controller-driver/csioptions/csioptions.go

```diff
@@ -76,3 +90,22 @@ func NewCSIOptions() *CSIOptions {
 	}
 	return &csioptions
 }
+
+// GetFssAddress TODO : Remove this function once the value of fss-address and fss-endpoint is passed from management-plane
```

Comment:
please remove comment related to management plane

Reply:
Done